recipes/arsenic/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "arsenic" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/openforcefield/openff-{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: f7bcada177e3dbe1486ffa4461184d045b48eb266379bc2a95b026fbe26363e1
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python>=3.6
+    - pip
+  run:
+    - networkx
+    - numpy
+    - matplotlib
+    - seaborn
+    - plotly
+    - python>=3.6
+    - scikit-learn
+    - sphinx
+    - sphinx_rtd_theme
```

Comment:
I don't think these are needed by users of the package, only for building the docs?
```suggestion
```